### DISCUSSION TITLE
Robomk works on checkmk cre 2.0 ?

### DISCUSSION THREAD

**Discussion by tjyang** [elabit/robotmk#180] [Q&A] [answered]
I am new cmk 2.0 user, not sure if robomk will work with CRE version ? 

From https://exchange.checkmk.com/p/robotmk, look like it only work with CEE version ? 
![image](https://user-images.githubusercontent.com/378638/145720371-b9dd6be6-0fd4-4588-855a-5600922c8e57.png)

**Comment by simonmeggle** [comment, marked as answer]
It should also work on CRE, but you will have to take care for the plugin
deployment and robotmk.yml  for yourself.
My strong recommendation is to take the free enterprise Version of checkmk
which you can use up to 25 hosts. I am sure if you see the added value of
the bakery, you won't write robotmk.yml by hand.
In any case, you will learn from the generated file.
Regards,
Simon

T.J. Yang ***@***.***> schrieb am So. 12. Dez. 2021 um 17:21:

> I am new cmk 2.0 user, not sure if robomk will work with CRE version ?
>
> From https://exchange.checkmk.com/p/robotmk, look like it only work with
> CEE version ?
> [image: image]
> <https://user-images.githubusercontent.com/378638/145720371-b9dd6be6-0fd4-4588-855a-5600922c8e57.png>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/simonmeggle/robotmk/discussions/180>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAOPHQSHUD5XWNXIT3X5FXTUQTDXXANCNFSM5J4NMZEQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 

______________________________________________



*ELABIT GmbH*
IT Monitoring | IT Automation | E2E Testing
Simon Meggle (CEO)
Gewerbering 3 | D-86504 Merching
Phone +49 (8233) 2171680 | Mobile +49 (179) 5447309
***@***.*** | www.elabit.de

**Reply by tjyang** [reply]
Thanks for the prompt reply. I will standup a POC box to run checkmk CEE free edition, to ease the learning cure of both robotmk+cmk 2.0 at the same time.

**Comment by simonmeggle** [comment]
...pleasure!

T.J. Yang ***@***.***> schrieb am So. 12. Dez. 2021 um 18:06:

> Thanks for the prompt reply. I will standup a POC box to run checkmk CEE
> free edition, to ease the learning cure of both robotmk+cmk 2.0 at the same
> time.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/simonmeggle/robotmk/discussions/180#discussioncomment-1794007>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAOPHQVFBESLD32LVNPBIVLUQTJAFANCNFSM5J4NMZEQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>
-- 

______________________________________________



*ELABIT GmbH*
IT Monitoring | IT Automation | E2E Testing
Simon Meggle (CEO)
Gewerbering 3 | D-86504 Merching
Phone +49 (8233) 2171680 | Mobile +49 (179) 5447309
***@***.*** | www.elabit.de
